dbt/main.py

```diff
@@ -38,4 +38,10 @@ def main(args=None):
 
     parsed = p.parse_args(args)
 
+    try:
+        proj.validate()
```

Comment:
I suggest we run this in the `project.read_project` function, which is the only place projects are created right now, so we know that if a project exists then it's also valid